### DISCUSSION TITLE
How to define the inside of spherical polygon for correction?

### DISCUSSION THREAD

**Discussion by rconde01** [boostorg/geometry#1061] [Q&A]
I'm trying to find a way to specify the inside of a polygon so that the results are correct when calling `correct`. For example, you might have a "cone" where the half angle is greater than 90 degrees. Currently this is corrected such that the axis of the code is flipped to the other side. Specifying a vector in the interior of the polygon should be able to resolve the ambiguity...but so far I don't see a way to do this.

**Comment by rconde01** [comment]
I think the answer is there is no way - so i'll convert this to an issue/feature-request

**Comment by vissarion** [comment]
There are two issues related to this discussion https://github.com/boostorg/geometry/issues/1097 and https://github.com/boostorg/geometry/issues/1075
